### DISCUSSION TITLE
Getting build error and GLIBC error

### DISCUSSION THREAD

**Discussion by CodeOnTheWall** [vercel/community#2306] [Bug Report]
im trying to deploy on vercel, here is my github: https://github.com/CodeOnTheWall/improved-portfolio and i keep getting these errors, and i dont know what is causing them: 

<html>
<body>

Running build in Washington, D.C., USA (East) – iad1
--
14:55:03.338 | Cloning github.com/CodeOnTheWall/improved-portfolio (Branch: main, Commit: 217eca8)
14:55:03.344 | Skipping build cache, deployment was triggered without cache.
14:55:03.672 | Cloning completed: 333.328ms
14:55:03.806 | Running "vercel build"
14:55:04.260 | Vercel CLI 29.1.1
14:55:04.506 | Installing dependencies...
14:55:16.532 |  
14:55:16.532 | added 329 packages in 12s
14:55:16.533 |  
14:55:16.533 | 117 packages are looking for funding
14:55:16.533 | run `npm fund` for details
14:55:16.554 | Detected Next.js version: 13.4.0
14:55:16.556 | Detected `package-lock.json` generated by npm 7+...
14:55:16.557 | Running "npm run build"
14:55:16.858 |  
14:55:16.858 | > kris-portfolio@0.1.0 build
14:55:16.858 | > next build
14:55:16.859 |  
14:55:16.867 | node: /lib64/libm.so.6: version `GLIBC_2.27' not found (required by node)
14:55:16.867 | node: /lib64/libc.so.6: version `GLIBC_2.28' not found (required by node)
14:55:16.887 | Error: Command "npm run build" exited with 1
14:55:17.547 | Deployment completed
14:55:17.118 | BUILD_UTILS_SPAWN_1: Command "npm run build" exited with 1

</body>
</html>

I have searched and searched, and I dont know what is causing them, i dont have any syntax error in my code, i can run npm run build in my terminal without issues. Please help

**Comment by CodeOnTheWall** [comment]
I had to make my node version 16 for this to work, for which i still have no idea why. If someone could please explain to me why this happened?

**Reply by fsa317** [reply]
I am having this same issue, when you changed node to 16 did you do that in package json or in vercel settings?

**Comment by CodeOnTheWall** [comment]
Hey mate, i just changed it in package Json. I'm not quite sure what exactly is interfering and causing issues nodejs 18+

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
